wave_matching/include/wave/matching/groundSegmentationParams.hpp

```diff
@@ -0,0 +1,78 @@
+#ifndef WAVE_GROUNDSEGMENTATIONPARAMS_H
+#define WAVE_GROUNDSEGMENTATIONPARAMS_H
+
+#include "wave/utils/config.hpp"
+#include "wave/utils/log.hpp"
+
+namespace wave {
+
+struct GroundSegmentationParams {
+    // variables
+    double rmax;         // max radius of point to consider
+    int max_bin_points;  // max number of points to consider per bin
+    int num_seed_points;
+
+    // for GP model
+    float p_l;  // length parameter, so how close the points have to be in the
+    // GP model to consider them correlated
+    float p_sf;      // scaling on the whole covariance function
+    float p_sn;      // the expected noise for the mode
+    float p_tmodel;  // the required confidence required in order to consider
+    // something ground
+    float p_tdata;  // scaled value that is required for a query point to be
+    // considered ground
+    float p_tg;  // ground height threshold
+
+    double robot_height;
+
+    // seeding parameters
+    double max_seed_range;   // meters
+    double max_seed_height;  // meters
+
+    int num_bins_a;
+    int num_bins_l;
+
+    // set default parameters
+    GroundSegmentationParams() {
+        rmax = 100;
```

Comment:
Move these assignments to default member initializers ([#2 here](http://en.cppreference.com/w/cpp/language/data_members#Member_initialization))